app/Http/Controllers/Areas/AreasController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Http\Controllers\Areas;
+
+use App\Models\DBC\Area\AreasAndZones as AreasAndZonesWotlk;
+use App\Models\WOD\DBC\Area\AreasAndZones as AreasAndZonesWod;
+use App\Exceptions\UnsupportedVersion;
+use App\Helpers\TCAPI;
+use Illuminate\Http\Request;
+
+use App\Http\Requests;
+use Illuminate\Routing\Controller;
+
+class AreasController extends Controller
+{
+    private $api;
+
+    /**
+     * AchievementsController constructor.
+     *
+     * @param TCAPI $api
+     */
+    public function __construct(TCAPI $api)
```

Comment:
I've learned a lot about Laravel and I know that it is possible to do things in different ways which are all correct.

I know what you have done with the TCAPI class but this way of initializing an object makes just sense for e.g. repositories. TCAPI delivers static content so why we don't make a singleton?